community/office-hours.md

```diff
@@ -12,6 +12,36 @@ Third Wednesday of every month, there are two sessions:
 Tune into the [Kubernetes YouTube Channel](https://www.youtube.com/c/KubernetesCommunity/live) to follow along. 
 You can post questions on the [#office-hours channel](https://kubernetes.slack.com/messages/office-hours) on Slack.
 
+### How it Works
+
+#### Bringing your own Stack Overflow Question
+
+If you submit a [SO question](https://stackoverflow.com/questions/tagged/kubernetes) we can prepare ahead of time and check important details. 
```

Comment:
based on text above seems I'm assuming #office-hours is the only place to submit questions?